Makefile

```diff
@@ -84,6 +84,15 @@ py-pb:
 		.;
 	cp -a output/bess_pb/. ${PTF_PB_DIR}
 
+.coverage:
+	rm -rf $(CURDIR)/.coverage
+	mkdir -p $(CURDIR)/.coverage
+
+test: .coverage
+	@echo
+	@echo "==> Running unit tests with coverage <=="
+	go test -race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -v $(CURDIR)/pfcpiface
```

Comment:
```suggestion
test: .coverage
	docker run --rm -v $(CURDIR):/upf-epc -w /upf-epc golang:latest \
		go test \
			-race \
			-coverprofile=.coverage/coverage-unit.txt \
			-covermode=atomic \
			-v \
			./...
```

Reply:
running go test [...] ./... would start also integration tests

Reply:
got it, then limiting it to `pfcpiface` is right for just unit tests.

Reply:
That's right. But I'm ok to run unit tests inside Docker.

Reply:
There is the small drawback of having to download the Go deps every run. But that could be addressed by creating a common go image.

```docker
FROM golang:latest as golang-build # we should tag this
WORKDIR /pfcpiface
RUN go mod download
```

Reply:
> That's right. But I'm ok to run unit tests inside Docker.

Great! Let's see if running in Docker fixes the error above, then merge if it does.